Documentation/Proposals/0000-proposal-template.md

```diff
@@ -0,0 +1,184 @@
+# Feature name
+
+* Proposal: [ST-NNNN](NNNN-filename.md)
```

Comment:
Since we use "swt"/"SWT" as our prefix in code, should we use it here?

Reply:
Sounds good